### ISSUE TITLE
inconsistent install path 

### ISSUE THREAD

**Issue by lockbox** [devkitPro/devkitarm-rules#9]
https://github.com/devkitPro/devkitarm-rules/blob/ab0f5c46edaf0d3e7d582bf0e66430dbde92a902/Makefile#L10-L12

when this gets called from [build_crtls.sh](https://github.com/devkitPro/buildscripts/blob/master/dkarm-eabi/scripts/build-crtls.sh) (which is called from [build_devkit.sh](https://github.com/devkitPro/buildscripts/blob/master/build-devkit.sh#L226)), regardless of what the actual top-level `$INSTALLDIR` is, the rules get copied to `/opt/devkitpro/devkitARM` as opposed to `$INSTALLDIR/devkitpro/devkitARM` (which then makes things explode when all our arm rules are "missing" in `/opt` instead of under `$DEVKITARM`)

TLDR;
- `$DESTDIR` is not set when this is invoked
- this ignores the top level installation prefix that is set in `build-devkit.sh`
- it results in install/builds failing due to none of the rules being where it expects.

What is the right way to fix this? I'm happy to do a PR, to solve my problem i just patched the `Makefile` to use `$INSTALLDIR` instead, but i'm not sure thats correct for all the automated `pacman` packaging etc.

**Comment by WinterMute** [comment]
Installing to other prefixes is not supported. Since moving to pacman for distribution we have standardised to /opt/devkitpro for all our packages. The environment variables are really legacy artifacts at this point.

https://github.com/devkitPro/buildscripts/blob/master/build-devkit.sh#L74-L86

If you're in a situation where you cannot access /opt/devkitpro then we recommend using the provided docker images instead. See https://hub.docker.com/repositories/devkitpro

**Comment by lockbox** [comment]
Fair enough, thanks 👍
